.ci.yaml

```diff
@@ -84,7 +84,7 @@ platform_properties:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:29.0"},
-          {"dependency": "chrome_and_driver", "version": "version:84"},
+          {"dependency": "chrome_and_driver", "version": "kkOzlCHhXTi3TpU7miUzqSOCVa_2RJVb09kfze0a6z0C"},
```

Comment:
Can we tag this cipd ref as version:96 to make it easier to audit?

Reply:
Do you have the command line to do this?

Reply:
`cipd set-tag flutter_internal/browsers/chrome/linux-amd64 -tag version:96 -version kkOzlCHhXTi3TpU7miUzqSOCVa_2RJVb09kfze0a6z0C`